transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/wizards/NewTransformationWizard.java

```diff
@@ -434,14 +432,16 @@ private String generateModel(String filePath,
             return getJavaQualifiedName(filePath, monitor);
         }
         case JSON: {
+            String className = classNameBuilder.toString().replace("_", "");
```

Comment:
After looking at our tests, we don't have enough in the test framework to test this; the code is on the UI side only and is not conducive to allowing for tests right now.  I already have an issue logged to address our lack in testability in the UI, so this will have to be addressed when that issue is addressed.